nix/lib/github/githubWebhook.nix

```diff
@@ -22,7 +22,7 @@ utils: lib: {
 
       echo null | jq --argjson body "$body" '[]
         | if $body.created or $body.deleted then . + [{"command":"UpdateJobsets"}] else . end
-        | if $body.deleted | not then . + [{"command":"EvaluateJobset","name":$body.ref|split("/")|.[2]}] else . end
+        | if $body.deleted | not then . + [{"command":"EvaluateJobset","name":$body.ref|split("/")|.[2:]|join("/")}] else . end
```

Comment:
Not related to that PR, but I find such jq expression without any space hard to read

Reply:
As mentionned in #20, there is a lot that can be done to refactor `mkGithubProject` and `mkGiteaProject` in a nice way. I will open a new PR once this one and #20 are merged :)